tools/pull-request-labeler/entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh -l
+/pull-request-labeler/node_modules/.bin/ts-node --project /pull-request-labeler/tsconfig.json /pull-request-labeler/src/main.ts
```

Comment:
can you use the npm script here? `npm run labeler`?

Reply:
same answer as https://github.com/dynatrace-oss/barista/pull/305#discussion_r355416685